kotlin-native/runtime/src/legacymm/cpp/Memory.cpp

```diff
@@ -252,7 +252,17 @@ class CycleDetector : private kotlin::Pinned, public KonanAllocatorAware {
 
 // TODO: can we pass this variable as an explicit argument?
 THREAD_LOCAL_VARIABLE MemoryState* memoryState = nullptr;
-THREAD_LOCAL_VARIABLE FrameOverlay* currentFrame = nullptr;
+
+#ifdef KONAN_WINDOWS
```

Comment:
We might also want to add a def here to disable it.
In my code im disabling emutls altogether, and without that performance is better with plain tls since it doesn't require any syscall.

Something like `DISABLE_CUSTOM_WIN32_TLS`?

Reply:
That would be great!
Similar defines typically use the `KONAN_` prefix.